### DISCUSSION TITLE
View ALL contacts across every alias

### DISCUSSION THREAD

**Discussion by markliederbach** [simple-login/app#928] [Feature Requests]
We can currently view all contacts for a single alias, but there's currently no way to see every contact regardless of the alias it belongs to.

I think it would be beneficial to rework the contact view to include each alias (maybe with the addition of a field to indicate which alias it belongs to). This view could also be useful for filtering on certain contacts, and even performing bulk actions such as DELETE (related request: https://github.com/simple-login/app/discussions/878).

**Comment by dwly26** [comment]
I agree. The view needs to show the alias it belongs to, and ideally the email address it is going to. There should be a way to filter this list as with the list of aliases. A use for this is to add these contacts to your address book - for use with your email program for example. After you get tens of aliases, it is laborious (and time-wasting) to have to click on every alias just to find out if it has a reverse alias in use, and to what contacts. Thanks for listening.
